tutorials/3d/using_transforms.rst

```diff
@@ -149,17 +149,21 @@ It is possible to rotate a transform, either by multiplying its basis by another
 .. tabs::
  .. code-tab:: gdscript GDScript
 
+    var axis = Vector3(1, 0, 0) # Or Vector3.RIGHT
+    var rotation_amount = 0.1
     # Rotate the transform about the X axis
-    transform.basis = Basis(Vector3(1, 0, 0), PI) * transform.basis
+    transform.basis = Basis(axis, rotation_amount) * transform.basis
     # shortened
-    transform.basis = transform.basis.rotated(Vector3(1, 0, 0), PI)
+    transform.basis = transform.basis.rotated(axis, rotation_amount)
 
  .. code-tab:: csharp
 
+    var axis = Vector3.Right
+    var rotation_amount = 0.1
     // rotate the transform about the X axis
```

Comment:
We should have consistency between GDScript and C#. Considering the semantics here are that we want to rotate around the X axis, using `Vector3(1, 0, 0)` makes sense.

Also, your C# code has some errors (some are style errors, some are important). Be sure to also fix the variable name on the other lines to match what I'm suggesting here, and update the GDScript comment to match.

```suggestion
    Vector3 axis = new Vector3(1, 0, 0); // Or Vector3.Right
    float rotationAmount = 0.1f;
    // Rotate the transform about the X axis by 0.1 radians.
```

Reply:
> We should have consistency between GDScript and C#. Considering the semantics here are that we want to rotate around the X axis, using Vector3(1, 0, 0) makes sense.

Great catch.

> Also, your C# code has some errors (some are style errors, some are important).

Thank you. I copied these from an earlier section as I'm unfamiliar with C#, but I must have mistakenly copied from a gdscript section. Thank you for catching that.

> Be sure to also fix the variable name on the other C# lines to match what I'm suggesting here, and update the GDScript comment to match.

I'm not 100% sure which lines you mean, but I'll check again now. Thank you again for catching my errors.

Reply:
@aaronfranke I have applied your commit to the branch and added one other one to update the remaining examples. I initially didn't do this as I wasn't sure if the change would even be accepted, but after seeing your thoughtful review, I've put in more effort.

Thank you for the fixes. If you would prefer that I do a fixup on my changes, remove the new changes, make any additional changes or squash all three commits into one (adding you as a co-author), just let me know.